pkg/occlient/occlient.go

```diff
@@ -248,6 +249,44 @@ func (c *Client) isLoggedIn() bool {
 	return true
 }
 
+// RunLogout logs out the current user from cluster
+func (c *Client) RunLogout() error {
+	output, err := c.userClient.Users().Get("~", metav1.GetOptions{})
```

Comment:
Won't this fail on Windows?

Reply:
I've not tested on windows yet, will find a windows machine and validate
but what made you think like that, can you please provide some insight?

Reply:
@syamgk Please test this PR on windows before merging